recipes/default.rb

```diff
@@ -25,39 +25,44 @@
 nslcd_enable = false
 case node['platform']
 when 'redhat', 'centos', 'scientific'
-  case node['platform_version'].to_i
-  when 7
-    sssd_action = 'install'
-    # CentOS 7 requires authconfig update to avoid bugs with multiple LDAP servers
-    authconfig_action = 'upgrade'
-    node.default['authconfig']['ldap']['packages'] = ['nss-pam-ldapd']
-  when 6
+  if node.key?('cloud') && node['cloud'].key?('provider') && node['cloud']['provider'].downcase == 'ec2'
```

Comment:
Yeah, the safe navigation would be more concise.

I'm not sure why we do ldap either, but I'm just preserving the amazon behavior.